libs/filamat/include/filamat/MaterialBuilder.h

```diff
@@ -499,6 +500,8 @@ class UTILS_PUBLIC MaterialBuilder : public MaterialBuilderBase {
     // Return true if:
     // The shader is syntactically and semantically valid
     bool findProperties() noexcept;
+    bool findProperties(filament::backend::ShaderType type,
```

Comment:
I'm not sure an overload is the clearest solution here. Maybe rename `findProperties()` to `findAllProperties()`? Maybe you have a better idea for a name.

Reply:
SG